zarr.html

```diff
@@ -0,0 +1,77 @@
+---
+layout: idr
+title: Zarr
+redirect_from:
+  - zarr/
+---
+
+        <!-- begin marketing header -->
+        <header class="marketing-hero">
+            <div class="row homepage text-center">
+
+                <div class="medium-12 columns">
+
+                    <h1 class="hero-main-header">Cloud access</h1>
+                    <p class="hero-subheader small-10 medium-10 large-10 small-offset-1 medium-offset-1 large-offset-1">
+                    The imaging data for some studies published in the
+                    Image Data Resource (IDR) is also available in a converted
+                    form on S3.
+                    </p>
+                </div>
+            </div>
+        </header>
+
+        <!-- begin Example section -->
+
+        <hr class="whitespace">
+        <div class="row">
+            <div class="small-10 small-centered medium-10 medium-centered columns">
+                <div class="row horizontal">
+                    <div>
+                        
+                        TODO: include zarr.md
+                        
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="small-12 small-centered medium-12 medium-centered columns">
+                <div class="row horizontal">
+                    <table>
+                        <thead>
+                            <th>Image</th>
+                            <th>Name</th>
+                            <th>Study</th>
+                            <th>X</th>
+                            <th>Y</th>
+                            <th>Z</th>
+                            <th>C</th>
+                            <th>T</th>
+                            <th title="Link to the data in ome-zarr format">ome-zarr</th>
+                            <th title="Available in all older versions since the listed version">Since</th>
+                        </thead>
+                        <tbody>
+                        {% for row in site.data.zarr %}
+                            <tr>
+                                <td><a href="/webclient/?show=image-{{ row.Image }}">{{ row.Image }}</a></td>
+                                <td><a href="/webclient/?show=image-{{ row.Image }}">{{ row.Name }}</a></td>
```

Comment:
What's the value of the duplicate hyperlink here?

Reply:
Don't know, that was @joshmoore's. Happy to remove one of them if preferred?

Reply:
So you can click on either the IDR or the name.

Reply:
Actually, with using the ID in the zarr link (below) we don't need to list the Image ID column either, so I can save space and remove that column.

Reply:
Ooops - sorry Josh, missed your reply there. I find it can be confusing if you have different text linking to the same thing as you end up clicking on both (or having to inspect them to realise they are the same).

Reply:
Then I'd err on the side of linking the ID rather than the name.